### PR TITLE
Adjust elasticsearch port to default again for Circle CI

### PR DESCRIPTION
**Pull request by bruderol**
@cgrossde here you are :-)

### CODE REVIEW

**Comment by cgrossde** [comment]
@bruderol Nice! Should we then merge https://github.com/scenarioo/scenarioo/pull/758 into develop?

File: docs/contribute/ci-server-setup/Elasticsearch-5.md [hunk 1]
```diff
@@ -4,7 +4,7 @@ Install and run using docker:
 
 ```
 docker pull docker.elastic.co/elasticsearch/elasticsearch:5.6.9
-docker run -d -name elasticsearch5 -p 9205:9200 -p 9305:9300 -e cluster.name=elasticsearch -e xpack.ml.enabled=false -e xpack.security.enabled=false docker.elastic.co/elasticsearch/elasticsearch:5.6.9 
+docker run -d --name elasticsearch5 -p 9205:9200 -p 9305:9300 -e cluster.name=elasticsearch -e xpack.ml.enabled=false -e xpack.security.enabled=false docker.elastic.co/elasticsearch/elasticsearch:5.6.9
```
Comment (cgrossde) [inline docs/contribute/ci-server-setup/Elasticsearch-5.md]: The port here is still `9205` and `9305`

Comment (bruderol) [reply docs/contribute/ci-server-setup/Elasticsearch-5.md]: ahrgh, yes, ok, I'll fix that as well

**Comment by bruderol** [comment]
will merge it now - check passed before already

**Pull request merged**